fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/client/HudRenderCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.event.client;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	/** Called after rendering the status effects overlay when rendering the hud. */
+	public static final Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class,
+		listeners -> delta -> {
+			for (HudRenderCallback event : listeners) {
+				event.render(delta);
+			}
+		}
+	);
+
+	/**
+	 * Called when the hud is rendered, which is displayed in game, in a world.
+	 *
+	 * @param delta The number of frames passed since the last full tick
+	 */
+	void render(float delta);
```

Comment:
Possibly include the `InGameHud` instance as a parameter `hud`?

Reply:
There isn't much in `InGameHud` that is useful anyways. Also it is just `MinecraftClient.getInstance().inGameHud`

Reply:
Someone may one to fire the callback for their own hud implementation that subclasses InGameHud. the fewer expectations that are made about the caller the better.

Reply:
The reason is that most methods in `InGameHud` are private (some are public), so I don't see people making subclasses for `InGameHud`. If anything they should mixin.

Reply:
Let's call this tickDelta (as appears in current yarn)